http-clients-api/src/main/java/com/palantir/remoting/clients/ClientConfig.java

```diff
@@ -87,13 +86,13 @@ public static ClientConfig fromServiceConfig(ServiceConfiguration serviceConfig)
 
         // timeouts
         if (serviceConfig.connectTimeout().isPresent()) {
-            clientConfig.connectTimeout(serviceConfig.connectTimeout().get().toMilliseconds(), TimeUnit.MILLISECONDS);
+            clientConfig.connectTimeout(serviceConfig.connectTimeout().get());
```

Comment:
sure

On Mon, Jul 25, 2016 at 4:53 PM Mark Elliot <notifications@github.com>
wrote:

> In
> http-clients-api/src/main/java/com/palantir/remoting/clients/ClientConfig.java
> <https://github.com/palantir/http-remoting/pull/183#discussion_r72166564>:
>
> > @@ -87,13 +86,13 @@ public static ClientConfig fromServiceConfig(ServiceConfiguration serviceConfig)
> >
> >          // timeouts
> >          if (serviceConfig.connectTimeout().isPresent()) {
> > -            clientConfig.connectTimeout(serviceConfig.connectTimeout().get().toMilliseconds(), TimeUnit.MILLISECONDS);
> > +            clientConfig.connectTimeout(serviceConfig.connectTimeout().get());
>
> Given that we're making a builder I think the builder should own this code?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/183/files/4a3fb8a3ee0417bef1314201a0e8a7803f8fa78b#r72166564>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwYHrqfmOXfUOUY5C4Qm3S76pW16jks5qZUyAgaJpZM4JUqGw>
> .
>